iothub/service/tests/RetryPolicies/IotHubServiceTestRetryPolicy.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Tests
+{
+    internal class IotHubServiceTestRetryPolicy : IotHubServiceRetryPolicyBase
```

Comment:
What's the purpose of this class?

Reply:
This is for unit test of `IotHubServiceRetryPolicyBase`. We also have similar testing class on the hub device side.

Reply:
Ah, please add a code comment.

My mistake, I think.